web/themes/custom/pineapple/pineapple.theme

```diff
@@ -55,6 +55,14 @@ function pineapple_preprocess_node(array &$variables) {
     $owner = $node->getOwner();
     $variables['owner'] = $owner;
   }
+
+  if ($node->getType() == 'article' || 'recipe') {
+
+    if (isset($node->_referringItem)) {
+      $parent = $node->_referringItem->getEntity()->getType();
```

Comment:
Parent variable should be named to represent its content/value

Reply:
done